totalRP3_Extended/aura/aura.xml

```diff
@@ -0,0 +1,244 @@
+﻿<Ui xmlns="http://www.blizzard.com/wow/ui/"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.blizzard.com/wow/ui/">
+	
+	<Frame name="TRP3_AuraFrame" parent="UIParent" enableMouse="true" movable="true" clampedToScreen="true">
```

Comment:
I don't know how I feel about extended-specific-things using "TRP3_" as a prefix. I feel as though it should be "TRP3_ExtendedAuraFrame" and so on to absolutely ensure no conflicts ever arise.

Reply:
We'll handle this ourselves if we find it's an issue - it's more systemic across the whole of extended anyway.